onedal/_base.py

```diff
@@ -16,23 +16,13 @@
 
 from abc import ABC
 
-from onedal import _backend
+from onedal import _backend as onedal_backend
 
-from ._policy import _get_policy
-
-
-def _get_backend(backend, module, submodule=None, method=None, *args, **kwargs):
-    result = getattr(backend, module)
-    if submodule:
-        result = getattr(result, submodule)
-    if method:
-        return getattr(result, method)(*args, **kwargs)
-    return result
+from .common._policy import _get_policy
 
 
 class BaseEstimator(ABC):
-    def _get_backend(self, module, submodule=None, method=None, *args, **kwargs):
-        return _get_backend(_backend, module, submodule, method, *args, **kwargs)
+    _backend = onedal_backend
```

Comment:
The idea here was not to have a link to the whole backend  lib as a property to the estimators, now it is broken.

Reply:
I guess we can discuss this, ideally we would alias in the necessary backend functions as part of the estimators in a case-by-case basis. i.e. for basic statistics we would do:
```python
from onedal._backend.basic_statistics import compute

class BasicStatistics(BaseEstimator, metaclass=ABCmeta):
      _compute = compute
```

Reply:
This would be very clear to the developer and the user where and what is being used, while also maintaining spmd inheritance simple.